common/flatpak-run.c

```diff
@@ -1828,20 +1828,25 @@ flatpak_run_apply_env_appid (FlatpakBwrap *bwrap,
   g_autoptr(GFile) app_dir_data = NULL;
   g_autoptr(GFile) app_dir_config = NULL;
   g_autoptr(GFile) app_dir_cache = NULL;
+  g_autoptr(GFile) app_dir_local = NULL;
 
   app_dir_data = g_file_get_child (app_dir, "data");
   app_dir_config = g_file_get_child (app_dir, "config");
   app_dir_cache = g_file_get_child (app_dir, "cache");
+  app_dir_cache = g_file_get_child (app_dir, "state");
```

Comment:
I guess you meant to initialize `app_dir_local` here?

Reply:
I did, yeah. Good catch.